talend-component-kit-intellij-plugin/.gitignore

```diff
@@ -0,0 +1,5 @@
+.gradle
+.idea/
```

Comment:
I think that some of those entries are already in top level .gitignore.

Reply:
I was just to make the developers life easier. 
I imported that folder as a separate project and my Idea didn't see some .gitignore exceptions from the parent root. 
(for example it didn't exclude "build" folder)